frontend/src/pages/dashboard.js

```diff
@@ -92,7 +92,8 @@ class Dashboard extends React.Component<Props> {
       .filter(x => this.state.filters.statuses.includes(x.status))
       .filter(x => this.state.filters.years.includes(x.year))
       .filter(x => !x.role.map(role => this.state.filters.roles.includes(role)).includes(false))
-      .filter(x => x.name.toLowerCase().includes(this.state.search.toLowerCase()))
+    console.log(this.state.filters.roles)
+    // .filter(x => x.name.toLowerCase().includes(this.state.search.toLowerCase()))
```

Comment:
do we need to keep this?

Reply:
woopsies

Reply:
the commented filter shouldn't be uncommented too... just fixed it